pyproject.toml

```diff
@@ -1,13 +1,16 @@
 [build-system]
 requires = ["setuptools",
-            "setuptools_scm",
+            "setuptools_scm>=6.2",
```

Comment:
Any reason for this version requirement ? This forces using the 2 most recent versions released 2 months ago: https://pypi.org/project/setuptools-scm/#history

Reply:
I assume because this was the version which added pyproject.toml support? but I didn't check, this is just what the scm docs say.

Reply:
I don't think there is any downside to being strict version wise here since it gets installed into the isolated build environment automatically anyway.

Reply:
> this is just what the scm docs say to do

Yeah, that's where I grabbed it from. I can try to see if any earlier version works.

Reply:
It looks like this can be replaced with `"setuptools_scm[toml]>=3.4"`
Later versions allow for the [toml] to be dropped.

Reply:
As @astrofrog  says, I don't think this is an issue. We run very modern Cython and other things in this list, I am happy to keep it where it is.

Reply:
I'm asking because recently they broke our CI (and many others) with their tomli dependency.
e.g. https://github.com/pypa/setuptools_scm/issues/608, https://github.com/pypa/setuptools_scm/issues/624
Indeed they recommend >=6.2 but all the versions between 6.1 and 6.3 have been yanked.
Bu thopefully should be fine :crossed_fingers:

Reply:
We can always hard pin a version if we don't trust them with releases?

Reply:
So merge with this version lower-bound? I guess if there's no version 6.2 it just goes up...